backend/vaa-strapi/eslint.config.mjs

```diff
@@ -0,0 +1,34 @@
+import { default as sharedConfig } from '@openvaa/shared-config/eslint';
```

Comment:
We're now importing the base settings from @openvaa/shared-config to all the subconfigurations.